matrix/main.py

```diff
@@ -39,6 +39,8 @@ def setup(matrix, args=None):
     parser.add_argument("-L", "--log-name", nargs="*")
     parser.add_argument("-f", "--log-filter", nargs="*")
     parser.add_argument("-s", "--skin", choices=("tui", "raw"), default="tui")
+    parser.add_argument("-x", "--xunit", default=None, metavar='FILENAME',
```

Comment:
might be better just to make this a skin type, no? Or are you thinking we'd want this available regardless of the on screen output type?